src/ar.erl

```diff
@@ -439,6 +440,7 @@ start(
 		false -> ok;
 		true  -> app_ipfs_daemon_server:start()
 	end,
+	ar_node:add_peers(Node, app_webhook:start(WebhookConfigs)),
```

Comment:
I'd say we should make it the core `ar_webhook` module for now as we can't move it out because it depends on the node config and needs to be hooked up here.

Reply:
Addressed in d950db75